SampleApp/SearchPinsViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SearchPinsViewController.swift
+//  ios-sdk
+//
+//  Created by Matt Smollinger on 1/6/17.
+//  Copyright © 2017 Mapzen. All rights reserved.
+//
+
+import UIKit
+import Pelias
+
+class SearchPinsViewController: MapViewController, UITextFieldDelegate {
+
+  @IBOutlet weak var searchField: UITextField!
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    self.loadScene("scene.yaml", apiKey: "mapzen-2qQR7SX")
```

Comment:
Sorry I haven't seen that this has already been discussed in https://github.com/mapzen/ios/pull/135.